.circleci/config.yml

```diff
@@ -425,6 +426,21 @@ jobs:
           name: Update RPM repository
           command: './scripts/build/update_repo/update-rpm.sh "oss" "$GPG_KEY_PASSWORD" "$CIRCLE_TAG" "dist"'
 
+  build-oss-msi:
+    docker:
+      - image: grafana/grafana-oss-ci-msi-build
+    working_directory: ~/repo
+    steps:
+      - checkout
+      - attach_workspace:
+          at: .
+      - run:
+          name: Create OSS MSI
+          command: |
+            ls -al /
+            ls -al /master
+            /master/ci-wrapper.sh
```

Comment:
You could use the CircleCI archive function to store the produced installer so that you can test it. We would remove this before merging though.

Reply:
i've added a store_artifacts step